src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -764,6 +759,14 @@ impl HummockVersion {
             &version_delta.state_table_info_delta,
             &changed_table_info,
         );
+
+        // The max_sub_level_id may have been increased, e.g. during commit_epoch or merge_group.
```

Comment:
We should max with `self.max_sub_level_id` and otherwise it will be reset to 0 when all data are clean.

Reply:
Fixed. Thanks.